.vib/mongodb/goss/mongodb.yaml

```diff
@@ -0,0 +1,17 @@
+file:
+  /opt/bitnami/mongodb/mongodb_get_source.txt:
+    mode: "0644"
+    exists: true
+    contains:
+      - official MongoDB Community
```

Comment:
Please, add back the whole string again. This is not something we get from upstream, but we add it ourselves